tests/tests_manifests_workflow/test_input_manifests.py

```diff
@@ -129,10 +129,16 @@ def test_add_to_cron(self, mock_open: MagicMock) -> None:
             f"TARGET_JOB_NAME=distribution-build-test;BUILD_PLATFORM=linux;BUILD_DISTRIBUTION=tar\n"
         )
 
-    def test_versionincrement_workflow(self) -> None:
+    def test_os_versionincrement_workflow(self) -> None:
         self.assertEqual(
-            InputManifests.versionincrement_workflow(),
-            os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "..", ".github", "workflows", "increment-plugin-versions.yml"))
```

Comment:
Wondering why didn't the test fail when we renamed the workflow. Any idea?

Reply:
The test ensures the the name of the file do match from the defined @classmethod, does not test if the file exists on the file system.
Example the test `test_os_versionincrement_workflow` asserts the file name same as defined in  @classmethod `os_versionincrement_workflow`.

Reply:
I see! Is there anyway we can add tests to make sure the file exists on the system?

Reply:
Added Sayali, thanks